### ISSUE TITLE
无法正确运行asrserver.py，报Data loss

### ISSUE THREAD

**Issue by bitsf** [nl8590687/ASRT_SpeechRecognition#243]
Hi, 我下载0.6.1 release 后，并且在我的Mac 上安装了最新的 tensorflow(2.5.0), keras(2.4.3), h5py(3.1.0) 等，在执行 `python asrserver.py` 后会提示错误如下
2021-06-02 14:54:15.312718: W tensorflow/core/util/tensor_slice_reader.cc:95] Could not open model_speech/speech_model251_e_0_step_625000.model: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
![image](https://user-images.githubusercontent.com/188115/120437271-7afe1900-c3b2-11eb-8208-3c28357249b5.png)

**Comment by nl8590687** [comment]
请看ASRT项目文档上写的内容，已经发布的v0.6.1版要用tf 1.15版本，keras 2.3.1
最新master的才可以用tf 2.x
